site/_plugins/download_filter.rb

```diff
@@ -43,27 +43,33 @@ def url_at_releases(platform, filename, ext)
       "#{RELEASES_URL_PREFIX}/manageiq-#{platform}-#{filename}.#{ext}"
     end
 
-    def file_size_from_url(url)
-      return "NA" if ENV["SKIP_FILE_SIZES"]
```

Comment:
I assumed you want to skip both size and md5 (avoiding http call) if you want to skip (not just one or the other) and changed the ENV name. I'm not sure if this ENV is currently used somewhere...